netkan/tests/mirrorer.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+
+from netkan.mirrorer import Mirrorer, CkanMirror
+
+
+# I don't know how to test Mirrorer, it needs a queue and archive.org access
+
+
+class TestCkanMirrorRedistributable(unittest.TestCase):
+
+    def setUp(self):
+        self.ckan_mirror = CkanMirror(contents = """{
+            "spec_version": "v1.4",
+            "identifier":   "AwesomeMod",
+            "name":         "Awesome Mod",
+            "abstract":     "A great mod",
+            "version":      "1.0.0",
+            "ksp_version":  "1.7.3",
+            "author":       [ "techman83", "DasSkelett", "politas" ],
+            "license":      [ "CC-BY-NC-SA-4.0", "GPL-3.0", "MIT" ],
+            "download_content_type": "application/zip",
+            "download_hash": {
+                "sha1": "DF564E21929EA07C624F822E5C43B7D0A3B0DBDF"
+            }
+        }""")
+
```

Comment:
The linter will probably complain about the extra spaces here and below :slightly_smiling_face:

Reply:
Indeed, it complained about several things...